app/controllers/sessions_controller.rb

```diff
@@ -28,8 +28,9 @@ def failure
 
   def setup
     if params.key?(:loa)
+      level = set_level(params[:loa])
       request.env['omniauth.strategy'].options[:authn_context] = [
-        "http://idmanagement.gov/ns/assurance/loa/#{params[:loa]}",
+        "http://idmanagement.gov/ns/assurance/loa/#{level}",
```

Comment:
This will not be recognized by the IdP since the authentication context URN is either LOA1 or LOA3. These need to be updated to follow a new schema for IAL. 

I propose `http://idmanagement.gov/ns/assurance/identity/<level>` for simplicity. Or, we could bring this up with the ICAM sub-committee to get a definitive answer. Either way, this will require a change within the IdP as well.